api/pkg/transformer/pipeline/util.go

```diff
@@ -16,6 +18,51 @@ func evalJSONPath(env *Environment, jsonPath string) (interface{}, error) {
 }
 
 func evalExpression(env *Environment, expression string) (interface{}, error) {
+	val, err := getVal(env, expression)
+	if err != nil {
+		return nil, err
+	}
+	switch exprVal := val.(type) {
+	case *operation.OperationNode:
+		val, err = exprVal.Execute()
+	case operation.OperationNode:
+		val, err = exprVal.Execute()
```

Comment:
nit: Can it do it like this?

```go
case  *operation.OperationNode, operation.OperationNode:
    val, err = exprVal.Execute()
```

Reply:
with the above `exprVal` type will be `interface` and can't call `Execute`